bundle/tests/scorecard/kuttl/test-cron-job/01-pods.yaml

```diff
@@ -18,7 +18,11 @@ spec:
       clusterName: my-cluster
       mode: none
     db:
-      image: "registry.redhat.io/rhel8/postgresql-12:1-36"
+      imageList: 
```

Comment:
are all these imageList changes related to the shared db stuff?

Reply:
No - it was just an opportunistic update